scripts/automation/README.md

```diff
@@ -0,0 +1,12 @@
+To run this, first create an environment file with the following and fill in the [BLANKS]:
```

Comment:
perhaps replace what is currently in the `fixtures` folder ?

not sure `scripts` is the right place

Reply:
fixed